scripts/populate_rcnotes_in_log_task.py

```diff
@@ -7,6 +7,7 @@
 # 'Nones' into specific column in a table
 
 import os.path as op
+from typing import Dict
```

Comment:
minor comment but where is Dict being used?

Reply:
fixed